code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -1114,3 +1114,83 @@
 		if(initial(liber.filterToxins))
 			liber.filterToxins = TRUE
 	L.metabolism_efficiency += metab_inibition
+
+/datum/reagent/toxin/radscorp
+	name = "Radscorpion Venom"
+	description = "Venom extracted from a radscorpion."
+	color = "#8C7918" // rgb: 140, 121, 24
+	taste_description = "sweetness"
+	taste_mult = 1.2
+	value = REAGENT_VALUE_COMMON //Encouraging people to mix toxins for reasons beyond harming each other or mixing reagents such as pen acid.
+	toxpwr = 1.3
+	ghoulfriendly = TRUE
+
+/datum/reagent/toxin/cazador
+	name = "Cazador Venom"
+	description = "Venom extracted from a cazador."
+	color = "#8C4C18" // rgb: 140, 76, 24
+	taste_description = "tanginess"
+	taste_mult = 1.2
+	value = REAGENT_VALUE_COMMON //Encouraging people to mix toxins for reasons beyond harming each other or mixing reagents such as pen acid.
+	toxpwr = 1.8
+	ghoulfriendly = TRUE
+
+/datum/reagent/toxin/silversting
+	name = "Silver Sting"
+	description = "A poison made from adding silver dust to radscorpion venom."
+	color = "#F0DE97" // rgb: 240, 222, 151
+	taste_description = "metallic sweetness"
+	taste_mult = 1.2
+	value = REAGENT_VALUE_COMMON //Encouraging people to mix toxins for reasons beyond harming each other or mixing reagents such as pen acid.
+	toxpwr = 1.5
+	ghoulfriendly = TRUE
+
+/datum/reagent/toxin/silversting/on_mob_life(mob/living/carbon/M)
+	M.adjustStaminaLoss(REM * data, 0)
+	data = max(data - 1, 1)
```

Comment:
here, data is 0, it was never given a value for this toxin, so it will not do any stamina damage